### PR TITLE
fix: [ORI-1671] Add method getDeposit, which exposes the deposit endpoint

### PR DESCRIPTION
**Pull request by avmturo**
- Add the getDeposit method to the sdk
- Add types for return value
- Add test for the getDeposit

### CODE REVIEW

File: test/sdk/e2e.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const expect = chai.expect;
 chai.use(chaiAsPromised);
 require('dotenv').config({ path: `${process.cwd()}/test/sdk/.env` });
 
+const ACCEPTANCE_CHAIN_ID = 80001;
+const ACCEPTANCE_NETWORK = 'Mumbai';
 describe('Original sdk e2e-method tests', async () => {
```
Comment (avmturo) [inline test/sdk/e2e.js:12]: Didn't feel this needed to be in secrets, as it's public knowledge and wont be changing. 
@don1989 wdyt?

Comment (don1989) [reply test/sdk/e2e.js:12]: Agreed

File: src/types.ts [hunk 2]
```diff
@@ -118,6 +118,13 @@ export type Burn = {
   uid: string;
 };
 
+export type DepositDetails = {
```
Comment (don1989) [inline src/types.ts:121]: This is what is returned from the endpoint right? If so, it should just be called Deposit. Like Burn, Asset, Transfer

Comment (avmturo) [reply src/types.ts:121]: Yeah, the whole naming convention around 'deposit' does seem a bit strange to me, as it's actually just returning user details. But will make this change

**Pull request merged**